packages/slides-frontend/src/containers/redux-store/DeckReducer/definitions.ts

```diff
@@ -36,18 +34,20 @@ export class Text extends baseItem {
       this.type = ItemTypes.TEXT
       this.Data = "<p></p>\n";
       this.Edit = false;
-      this.Size = { width: 500, height: 80 };
+      this.Position= {x: 0.30, y: 0.40};
+      this.Size = { width: 0.25, height: 0.05 };
```

Comment:
You are using `{ x }` and `{x}`. You should just stick to one of them :P

Reply:
:+1: